.cirrus.yml

```
@@ -852,6 +852,33 @@ buildah_bud_test_task:
     main_script: *main
     always: *int_logs_artifacts
 
+rootless_buildah_bud_test_task:
+    name: *std_name_fmt
+    alias: rootless_buildah_bud_test
+    # Docs: ./contrib/cirrus/CIModes.md
+    only_if: $CIRRUS_CRON == 'treadmill'
+    depends_on:
+        - build
+        - rootless_integration_test
+    env:
+        TEST_FLAVOR: bud
+        DISTRO_NV: ${FEDORA_NAME}
+        # Not used here, is used in other tasks
+        VM_IMAGE_NAME: ${FEDORA_CACHE_IMAGE_NAME}
+        CTR_FQIN: ${FEDORA_CONTAINER_FQIN}
+        CI_DESIRED_NETWORK: netavark
+        PRIV_NAME: rootless
```

```suggestion
    env:
        <<: *stdenvars
        TEST_FLAVOR: bud
        PRIV_NAME: rootless
```

I think this de-duplicates it correctly, but please double-check.  I'm assuming having `$CI_DESIRED_RUNTIME=crun` set is not a problem.

---

Done, and I also refactored the `bud` task which is where I stole this one. Thanks for catching that. (I have very little confidence that I did the magic `<<:*` emoticon correctly, though, but if it runs in CI I will assume it's OK).

---

Thanks.  Looks correct to me.